dev-docs/bidders/alvads.md

```diff
@@ -0,0 +1,152 @@
+---
+layout: bidder
+title: Alva
+description: Prebid Alva Bidder Adapter
+pbjs: true
+pbs_app_supported: false
+biddercode: alvads
+media_types: banner, video
+gdpr_supported: true
```

Comment:
`gdpr_supported` is not a valid meta data field. 

You can claim `tcfeu_supported: true` , but you need to provide your `gvl_id` as meta data as well.

Please add as much meta information as possible. See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter for a full list.

Especially

```
safeframes_ok: true/false
deals_supported: true/false
floors_supported: true/false
fpd_supported: true/false
prebid_member: true/false
multiformat_supported: will-bid-on-any, will-bid-on-one, will-not-bid
ortb_blocking_supported: true/partial/false
```

help publishers vet your adapter